### ISSUE TITLE
WIOD Release 2016 compatibility

### ISSUE THREAD

**Issue by JNS728** [konstantinstadler/pymrio#68]
The `parse_wiod` function seems not to be compatible with the WIOD 2016 release anymore. The reason is (or one of the reasons?) that the WIOD 2016 data is provided as `xlsb` file and not as `xlsx` file:

```Python
def parse_wiod(path, year=None, names=("isic", "c_codes"), popvector=None):
    """Parse the wiod source files for the IOSystem
    WIOD provides the MRIO tables in excel - format (xlsx) at
    http://www.wiod.org/new_site/database/wiots.htm (release November 2013).
    To use WIOD in pymrio these (for the year of analysis) must be downloaded.
    The interindustry matrix of these files gets parsed in IOSystem.Z, the
    additional information is included as factor_input extension (value
    added,...)

...
```

Anyway, I guess this is not a very pressing issue since other databases currently seem more attractive to work with. I jut wanted to let you know.

**Comment by konstantinstadler** [comment]
Yes, I am aware of that. When I last updated there was no xlsb reader for python available, I believe that has changed. I might revisit this at some point, however currently I don't have any use for the 2016 release. Let me know if you want to work on that

**Comment by JNS728** [comment]
Pandas can now handle xlsb: https://stackoverflow.com/a/60019546/5696601
I currently work with Exiobase. I'll let you know if I can make a _meaningful_ contribution.
